py/server/deephaven/barrage.py

```diff
@@ -116,6 +117,7 @@ def barrage_session(host: str,
                     auth_token: str = "",
                     use_tls: bool = False,
                     tls_root_certs: bytes = None,
+                    extra_headers: Dict[str, str] = None
```

Comment:
In `pydeephaven.Session.__init__`,  the types are `bytes`. Should we make them consistent?

Reply:
For whatever reason, the `Session` code path has more low level gRPC calls that accepted the `bytes` values. I decided to stick with higher level client config calls that accept `str`. I don't see a reason to force consistency here, but I'll change if Core devs think it's better.

Reply:
I am happy with keeping it as is.